telescopebe/backend.cpp

```diff
@@ -0,0 +1,23 @@
+#include <QtQml>
+#include <QtQml/QQmlContext>
+#include "backend.h"
+#include "../starfish/servicecontrol.h"
+#include "../starfish/watch.h"
+#include "../starfish/watches.h"
```

Comment:
Those few classes should probably be factorized into a common/ top level directory.

Reply:
Sure